ChustaSoft.GamersPlatformUtils.Domain.Xbox/Implementations/XboxBusiness.cs

```diff
@@ -1,31 +1,24 @@
 ﻿using ChustaSoft.GamersPlatformUtils.Abstractions;
 using System;
 using System.Collections.Generic;
-using System.Threading.Tasks;
-using System.Linq;
 using System.IO;
-using ChustaSoft.GamersPlatformUtils.Domain.Constants;
+using System.Linq;
+using System.Threading.Tasks;
 
-namespace ChustaSoft.GamersPlatformUtils.Domain.Implementations
+namespace ChustaSoft.GamersPlatformUtils.Domain
 {
-    public class XboxBusiness : PlatformBase, ILinkFinder
+    public class XboxBusiness : PlatformBase, IXboxBusiness, ILinkFinder
```

Comment:
As IXboxBusiness is now inheriting from ILinkFinder, should we remove ILinkFinder from here?

Reply:
Yes, it is not mandatory but seems to be a good practice (Check sdk's implementations like Dictionary or List do the same) 

This is because implementation of each contract has different purposes: IXXXBusiness is for DI purposes, the others are in the end  clearly defining on the implementation which contracts must implement.

However as it is not really mandatory, if you guys feel more comfortable we can remove it, but but the moment I tried to follow the same rules that the sdk is doing

Reply:
nice explanation! thanks so much! ok for me then.